packages/flutter/lib/src/services/text_input.dart

```diff
@@ -484,11 +495,24 @@ class TextInputConfiguration {
   /// Defaults to false.
   final bool readOnly;
 
+  @Deprecated(
+    'use obscure instead.'
+  )
   /// Whether to hide the text being edited (e.g., for passwords).
   ///
   /// Defaults to false.
   final bool obscureText;
 
+  @Deprecated(
+    'use obscure instead.'
+  )
+
+  /// Whether or not the last character entered is shown before
+  /// being obscured.
```

Comment:
This could probably use the flutter.widgets.editableText.obscureTextBehavior template.